### DISCUSSION TITLE
Results won't match with https://comfort.cbe.berkeley.edu/

### DISCUSSION THREAD

**Discussion by bzanchet** [CenterForTheBuiltEnvironment/pythermalcomfort#98] [General]
Hey there! I'm trying to calculate the PMV using the `pythermalcomfort.models.pmv` function but the results seem to disagree with the results from the website.

Am I missing something? Is there a reason for this difference? (they seem to differ even more with higher air speeds)

```python
from pythermalcomfort.models import pmv
from pythermalcomfort.utilities import v_relative, clo_dynamic

tdb = 25
tr = 27
v = 0.3
rh = 50
met = 1.3
clo = 0.96

# calculate relative air speed
v_r = v_relative(v=v, met=met)

# calculate dynamic clothing
clo_d = clo_dynamic(clo=clo, met=met)

results = pmv(tdb=tdb, tr=tr, vr=v_r, rh=rh, met=met, clo=clo_d, stardard='ASHRAE', airspeed_control=False)
print(f"air speed: {v_r}")
print(f"dynamic clothing: {clo_d}")
print(f"PMV: {results}")
```
> air speed: 0.39
> dynamic clothing: 0.871
> PMV: 0.63

vs the website:

![image](https://github.com/CenterForTheBuiltEnvironment/pythermalcomfort/assets/8005/0de49af7-969d-40f5-b5eb-89973709d0c6)

**Comment by FedericoTartarini** [comment]
Yes, the results are different since you misspelled the word `standard`. 

I will look into updating the python function to ensure that if a wrong input is provided you get an error.

**Reply by bzanchet** [reply]
🤦 x 10

sorry :(

**Reply by FedericoTartarini** [reply]
No worries it can happen 🙂. However, that was useful since it made me realise that package should return an error in that case. I then published a new version of pythermalcomfort.
